src/scss/themes/_full-bleed-image.scss

```diff
@@ -18,11 +18,6 @@
 			flex-direction: column;
 			justify-content: center;
 			padding: 20px;
-			min-height: 350px;
```

Comment:
I expect we still want this to keep the headline area of the topper tall on mobile

Reply:
Cool - updated